web3.md

```diff
@@ -110,15 +110,14 @@ The `blockList` cell stores a list of previous blocks and network states.
     rule #getAccountFromBlockchainItem ( { <network> <accounts> (<account> <acctID> ACCT </acctID> ACCOUNTDATA </account>) ... </accounts>  ... </network> | _ } , ACCT ) => <account> <acctID> ACCT </acctID> ACCOUNTDATA </account>
     rule #getAccountFromBlockchainItem(_, _) => .AccountItem [owise]
 
-    syntax BlockIdentifier ::= Int | String
- // ---------------------------------------
-
     syntax BlockIdentifier ::= #parseBlockIdentifier ( String ) [function]
  // ----------------------------------------------------------------------
     rule #parseBlockIdentifier(TAG) => TAG
-      requires TAG ==String "earliest"
-        orBool TAG ==String "latest"
-        orBool TAG ==String "pending"
+      requires TAG ==String "pending"
+
+    rule #parseBlockIdentifier("earliest")  => 0
+    rule [[ #parseBlockIdentifier("latest") => size(BLOCKLIST) -Int 1 ]]
+         <blockList> BLOCKLIST </blockList>
```

Comment:
If we just use `-1` we can avoid a contextual function here, which we're trying to do.